docs/contributing.md

```diff
@@ -455,6 +455,16 @@ you can submit a PR based on the `gh-pages` bookmark of
  The `.rs` files generated from `.proto` files are included in the repository,
  and there is a GitHub CI check that will complain if they do not match.
 
+## Logging
+
+You can print internal jj logs using `JJ_LOG`. It acts like the `RUST_LOG`
+environment variable, frequent in Rust codebases, and accepts one or more
+[directives]. You can also use the debug `--debug` global option that sets
+`debug` log level for all targets by default. `JJ_LOG` is still respected when
+using `--debug`.
+
+[directives]: https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#directives
```

Comment:
```suggestion
You can print internal jj logs using `JJ_LOG`. It acts like [the `RUST_LOG`
environment variable][rust-log-env-var] and accepts one or more
[directives]. You can also use the `--debug` global option that sets
`debug` log level for all targets by default. `JJ_LOG` is still respected when
using `--debug`.

[rust-log-env-var]: https://docs.rs/env_logger/latest/env_logger/#enabling-logging
[directives]: https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#directives
```

suggestion (non-blocking): I think the `EnvLogger` page on `RUST_LOG` is helpful since it has several concrete examples, so I added a link there as well.
suggestion (non-blocking): I changed "debug `--debug`" to just "`--debug`".
comment: The commit message says that it documents the relation of `JJ_TRACE`, but I don't think it mentions `JJ_TRACE` here? We could add `JJ_TRACE` to the discussion or just remove that from the commit message.

Reply:
- `tracing-subscriber` documentation also has a few examples, adding the `env_logger` link feels kind of like a duplicate to me.
- debug repetition is just a typo, good catch
- `JJ_TRACE` is explained just below in the "Profiling" section of "Contributing". I originally took the issue as is in the commit message